internal/db/schema/migrations/oss/postgres/78/01_server_worker_local_storage_state.up.sql

```diff
@@ -0,0 +1,86 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+
+create table server_worker_local_storage_state_state_enm (
+  state text primary key
+  constraint only_predefined_local_storage_states_allowed
+    check (
+      state in (
+        'available',
+        'lowStorage',
+        'criticallyLow',
+        'outOfStorage',
+        'notConfigured',
+        'unknown'
+      )
+    )
+);
+comment on table server_worker_local_storage_state_state_enm is
+  'server_worker_local_storage_state_state_enm is an enumeration table for worker local storage states.';
+
+insert into server_worker_local_storage_state_state_enm (state) values
+  ('available'),
+  ('lowStorage'),
+  ('criticallyLow'),
+  ('outOfStorage'),
+  ('notConfigured'),
+  ('unknown');
+
+alter table server_worker
+  add column local_storage_state text not null default 'available'
```

Comment:
I was trying to decide on the best possible default value for this when we upgrade a Boundary setup and then migration happens.

Since we decided on only picking workers for session recording that have an "available" local storage state, workers running on 0.13 will start off as "not configured" or "unknown" and will never be updated since nothing will be reporting the local storage state. 

In order not to break older workers/setups I wasn't too sure what the default should be. I left it as "available" so as not to break setups after migration.